code/modules/cargo/packs/misc.dm

```diff
@@ -214,10 +241,48 @@
 	for(var/i in 1 to 9)
 		new /obj/item/coin/silver(.)
 
+/datum/supply_pack/misc/randomised/dirtymags
+	name = "Dirty Magazines"
+	desc = "Get your mind out of the gutter operative, you have work to do. Three items per order. Possible Results: .357 Speedloaders, Kitchen Gun Mags, Stetchkin Mags."
+	hidden = TRUE
+	cost = 12000
+	var/num_contained = 3 
+	contains = list(/obj/item/ammo_box/a357,
+					/obj/item/ammo_box/a357,
+					/obj/item/ammo_box/a357,
+					/obj/item/ammo_box/magazine/pistolm9mm,
+					/obj/item/ammo_box/magazine/pistolm9mm,
+					/obj/item/ammo_box/magazine/pistolm9mm,
+					/obj/item/ammo_box/magazine/m45/kitchengun,
+					/obj/item/ammo_box/magazine/m45/kitchengun)
+	crate_name = "crate"
+
 //////////////////////////////////////////////////////////////////////////////
-//////////////////////////////// Misc Supplies ///////////////////////////////
+///////////////////////////////// Misc Supplies //////////////////////////////
 //////////////////////////////////////////////////////////////////////////////
 
+/datum/supply_pack/misc/candles
+	name = "Candle Crate"
+	desc = "Set up a romantic dinner or host a séance with these extra candles and crayons."
+	cost = 850
+	contains = list(/obj/item/storage/fancy/candle_box,
+					/obj/item/storage/fancy/candle_box,
+					/obj/item/storage/crayons)
+	crate_name = "candle crate"
+
+/datum/supply_pack/misc/interrogation
+	name = "CBT Interrogation Crate"
+	desc = "CBT is a special term used to describe an intricate form of information extraction. We can't tell you how to use this, but we can give you the tools required to get what you need."
+	contraband = TRUE
+	cost = 2400
+	contains = list(/mob/living/simple_animal/chicken,
+					/obj/item/toy/beach_ball/holoball,
+					/obj/item/melee/baton/cattleprod,
+					/obj/item/soap/deluxe,
+					/obj/item/stock_parts/cell/high)
+	crate_name = "interrogation crate"
+	crate_type = /obj/structure/closet/crate/large
+
```

Comment:
the exact way you feel about this being removed is how maintainers are going to feel about trying to just slap it back in in another pr randomly

Reply:
It's not random, several people have told me they found it funny, and asked to have it moved to contraband because having jokes in emag makes it hard to order.